tests/unit/kernel/CombinedKernel_unittest.cc

```diff
@@ -37,6 +42,135 @@ TEST(CombinedKernelTest,test_array_operations)
 	SG_UNREF(combined);
 }
 
+TEST(CombinedKernelTest, test_subset_mixed)
+{
+
+	CMeanShiftDataGenerator* gen = new CMeanShiftDataGenerator(0, 2);
```

Comment:
if you use `auto gen = some<CMeanShiftDataGenerator>(0,2);`, then you dont need to `SG_UNREF` later on